### DISCUSSION TITLE
Checkboxes in Dark theme not really visible

### DISCUSSION THREAD

**Discussion by daizuke123** [community/community#69058] [Accessibility] [answered]
I have been using issues to track development effort, with checkboxes to track progress. And I use Dark theme to reflect desktop settings. Sadly those two don't seem to blend.

**Comment by rnldvrgl** [comment, marked as answer]
Adjust Checkbox Color:
Check if the platform or tool you're using allows you to customize the color of checkboxes. You might be able to choose a color that works better with your dark theme. Consider using a contrasting color that stands out.

Increase Contrast:
Dark themes often have lower contrast, which can make checkboxes harder to see. Try increasing the contrast between the checkbox itself and its background. This can be done by adjusting the background color or the checkbox border color.

**Comment by Myndex** [comment]
> ..._Dark themes often have lower contrast, which can make checkboxes harder to see_. ...

However, dark themes shouldn't necessarrly have actually lower contrast, but WCAG 2.x math is wrong, expecially for darker colors. There are perceptually uniform methods and guidelines that should be used instead when working with dark mode.

The only dark mode here I can even use is the High Contrast one. The others might as well be invisible, because the are to me at least.

**Comment by cehfisher** [comment]
Hello @daizuke123 - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again! (cc @queenofcorgis)
